### DISCUSSION TITLE
How to generate different microstructuresfor same volume without changing the statistics?

### DISCUSSION THREAD

**Discussion by arunbabuktkm** [neperfepx/neper#620] [Q&A] [answered]
I cannot generate different microstructure realisations (or microstructures) for the given mean and standard deviations whenever I attempt to synthesise microstructure for a given volume. Each time the identical microstructure is generated.

**Comment by rquey** [comment, marked as answer]
Use [-id](https://neper.info/doc/neper_t.html#cmdoption-id). You may also want to read the tutorial about [Generating and Meshing a Simple Tessellation](https://neper.info/doc/tutorials/simple_model.html).
